### ISSUE TITLE
Clean cookie code

### ISSUE THREAD

**Issue by bradmorton1** [nhsuk/nhsuk-service-manual#82]
For a short term solution, the cookie consent code was taken directly from other nhs.uk solutions, with a simpler design applied to it. 

As the design is simpler and a user controls cookies with the browser (rather than in site settings), a lot of the code isn't required and can be simplified. 

The code is also currently compiled by webpack and isn't very readable in this repo. It needs to be changed to the pre-compiled code and assets.

**Comment by AdamChrimes** [comment]
@bradmorton1 are you aware of these errors relating to the cookie banner javascript?

Chrome console
```
Uncaught SyntaxError: Unexpected token e in JSON at position 1
    at JSON.parse (<anonymous>)
    at getCookie (cookieconsent.js:40)
    at checkCookie (cookieconsent.js:143)
```

Build warnings
```
[16:36:53] Finished 'minifyJS' after 1.56 s
app/scripts/cookieConsent.js (269:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (283:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (299:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (315:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (331:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (345:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (361:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
app/scripts/cookieConsent.js (375:6) Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
```

**Comment by bradmorton1** [comment]
Yeah, the first error is present because it's dropping the cookie but doesn't need to contain most of the info in the cookie. The 2nd lot is because it's using compiled webpack code, and we probably don't want that. Can make improvements some time this week

**Comment by AdamChrimes** [comment]
This was fixed in release 1.0.3
